### ISSUE TITLE
Calypso rerouting to wp-admin for image editing and then failing to load preview of image

### ISSUE THREAD

**Issue by xpurichan** [Automattic/wp-calypso#23706]
#### Steps to reproduce
1. Starting at URL: lillytest123456.blog (or **any** non-AT WP.com site)
2. My Sites > Customize > Colors & Backgrounds
3. Select image for background, [screenshot](https://snag.gy/EMhJLA.jpg)
4. Select Edit image on the right.
5. Calypso rerouted to wp-admin instead of going to the Calypso image editor
6. When following these steps the image preview fails to load, [screenshot](https://snag.gy/fPY3z0.jpg)

#### What I expected
-For the customizer to route to the calypso image editor
-For the image preview to load

#### What happened instead
- The customizer routed to wp-admin
- The image preview did not load

#### Browser / OS version
Chrome 65.0.3325.181 
Firefox 59

#### Context / Source
Just playing around and teaching a friend WordPress so they can apply. :)

## This image preview in wp-admin **will** load if accessed from:
1. lillytest123456.blog/wp-admin
2. Go to Media
3. Select image, then select Edit [screenshot](https://snag.gy/hgqpmB.jpg)

or from:

1. lillytest123456.blog/wp-admin
2. Go to Appearance > Customize > Colors & Background (wp-admin routes to calypso, as expected)
3. Select image for background, [screenshot](https://snag.gy/EMhJLA.jpg)
4. Select Edit image on the right
5. Calypso routed back to wp-admin and loads the image, [screenshot](https://snag.gy/I0bWhZ.jpg)

**Comment by rachelmcr** [comment]
I reproduced this issue with the steps above.

The customizer itself isn't part of Calypso, so it's loaded in an iframe. When you select "Edit Image" from within the customizer, it opens WP Admin but still includes a `frame-nonce` parameter, which results in a 400 error on the ajax request with the `imgedit-preview` action. When I manually removed the iframe nonce from the URL the WP Admin image editor loaded as expected with the image.
